kafka-connect-bloomberg/src/main/scala/com/datamountaineer/streamreactor/connect/bloomberg/BloombergSourceConfigConstants.scala

```diff
@@ -0,0 +1,55 @@
+package com.datamountaineer.streamreactor.connect.bloomberg
+
+object BloombergSourceConfigConstants {
+
+  val SERVER_HOST = "connect.bloomberg.server.host"
+  val SERVER_HOST_DOC = "The hostname running the bloomberg service"
+
+  val SERVER_PORT = "connect.bloomberg.server.port"
+  val SERVER_PORT_DOC = "The port on which the bloomberg service runs (8124-is the default)"
+
+  val SERVICE_URI = "connect.bloomberg.service.uri"
+  val SERVICE_URI_DOC = "The Bloomberg service type: Market Data(//blp/mkdata);Reference Data(//blp/refdata)"
+
+  val AUTHENTICATION_MODE = "connect.bloomberg.authentication.mode"
+  val AUTHENTICATION_MODE_DOC = "Optional parameter setting how the authentication should be done. " +
+    "It can be APPLICATION_ONLY or USER_AND_APPLICATION. Follow the Bloomberg API documentation for how to configure this"
+
+  val SUBSCRIPTIONS = "connect.bloomberg.subscriptions"
+  val SUBSCRIPTION_DOC = "Provides the list of securities and the fields to subscribe to. " +
+    "Example: \"AAPL US Equity:LAST_PRICE,BID,ASK;IBM US Equity:BID,ASK,HIGH,LOW,OPEN\""
+
+  val KAFKA_TOPIC: String = "connect.bloomberg.kafka.topic"
+  val KAFKA_TOPIC_DOC: String = "The name of the kafka topic on which the data from Bloomberg will be sent."
+
+  val BUFFER_SIZE = "connect.bloomberg.buffer.size"
+  val BUFFER_SIZE_DOC = "Specifies how big is the queue to hold the updates received from Bloomberg. If the buffer is full" +
+    "it won't accept new items until it is drained."
+
+  val PAYLOAD_TYPE = "connect.bloomberg.payload.type"
+  val PAYLOAD_TYPE_DOC = "Specifies the way the information is serialized and sent over kafka. " +
+    "There are two modes supported: json(default) and avro."
+
+  val BloombergServicesUris = Set("//blp/mkdata", "//blp/refdata")
+  val PayloadTypes = Set("json", "avro")
+
+
+  val EXPORT_ROUTE_QUERY = "connect.hbase.sink.kcql"
+  val EXPORT_ROUTE_QUERY_DOC = ""
+
+  val ERROR_POLICY = "connect.bloomberg.error.policy"
+  val ERROR_POLICY_DOC = "Specifies the action to be taken if an error occurs while inserting the data.\n" +
```

Comment:
I have been replacing this with the prettier version of multi-line strings:
"""
|asdasd
|asdasd
""".stripMargin